src/components/toolbar.tsx

```diff
@@ -31,11 +31,31 @@ export class ToolbarComponent extends BaseComponent<IProps, {}> {
     };
     return (
       <div className="toolbar">
-        <div className="tool select" title="Select" onClick={handleClickTool("select")}>↖</div>
-        <div className="tool text" title="Text" onClick={handleClickTool("text")}>T</div>
-        <div className="tool geometry" title="Geometry" onClick={handleClickTool("geometry")}/>
-        <div className="tool image" title="Image" onClick={handleClickTool("image")}/>
-        <div className="tool delete" title="Delete" onClick={handleClickTool("delete")}>{"\u274c"}</div>
+        <div className="tool select" title="Select" onClick={handleClickTool("select")}>
+          <svg className={`icon icon-select-tool`}>
+            <use xlinkHref={`#icon-select-tool`} />
+          </svg>
+        </div>
+        <div className="tool text" title="Text" onClick={handleClickTool("text")}>
+          <svg className={`icon icon-text-tool`}>
+            <use xlinkHref={`#icon-text-tool`} />
+          </svg>
+        </div>
+        <div className="tool geometry" title="Geometry" onClick={handleClickTool("geometry")}>
+          <svg className={`icon icon-geometry-tool`}>
+            <use xlinkHref={`#icon-geometry-tool`} />
+          </svg>
+        </div>
+        <div className="tool image" title="Image" onClick={handleClickTool("image")}>
+        <svg className={`icon icon-image-tool`}>
+            <use xlinkHref={`#icon-image-tool`} />
+          </svg>
+        </div>
+        <div className="tool delete" title="Delete" onClick={handleClickTool("delete")}>
+          <svg className={`icon icon-delete-tool`}>
+            <use xlinkHref={`#icon-delete-tool`} />
+          </svg>
+        </div>
```

Comment:
Should have a utility for creating SVGs...

Reply:
True, I just don't like having to repeat the `className` and `link` all over the place. Maybe this is fine, though

Reply:
That's what I mean. If you define a React component named SvgIcon that takes "icon-delete-tool" as a prop, then you can put it in the className and the xlinkHref and wherever else you need it, and the implementation is a three-line render function.

Reply:
OH gotcha, yes...I just don't really want to replace every SVG for this PR but I will add a chore 😬

Reply:
You're the one that suggested there should be a utility... 😉